content/links.md

```diff
@@ -408,3 +471,52 @@ instance_groups:
       - port
       - url
 ```
+___
+
+## Possible Link Scenarios
+
+The rules for uniqueness are described in the tables below:
+
+Providers can be specified in any way you want as long as it is **not** consumed.
+
+| Provider Name | Provider Type | Consumer Type | Allowed |
+|---------------|---------------|---------------|---------|
+| Same          | Same          | No consumer   | True    |
+| Same          | Different     | No consumer   | True    |
+| Different     | Same          | No consumer   | True    |
+| Different     | Different     | No consumer   | True    |
+
+Providers with different names or types can always be **explicitly** consumed without conflicts.
```

Comment:
also unclear to me what is the intention of this statement ?

Reply:
Updated with more context and suggestions for resolving issues.